src/lib/background/session/storage.js

```diff
@@ -148,25 +148,60 @@ exports.removePage = removePage;
  * Update one or more tab page properties
  * @param {number} tabId
  * @param {Object} tabUpdate
- * @param {string} tabUpdate.title
- * @param {string} tabUpdate.url
+ * @param {string} [tabUpdate.title]
+ * @param {string} [tabUpdate.url]
```

Comment:
looks like we're adding a `isAudio` option to `tabUpdate` here, so we should add a comment about it.

Reply:
Done. You're right, I missed that.